wtforms/fields/core.py

```diff
@@ -940,11 +940,16 @@ def validate(self, form, extra_validators=tuple()):
         for subfield in self.entries:
             if not subfield.validate(form):
                 self.errors.append(subfield.errors)
+            else:
+                self.errors.append(None)
+
+        if all(x is None for x in self.errors):
```

Comment:
Use `all(x for x in self.errors)` since we're referencing the lists instead of `None`.